.gitignore

```diff
@@ -7,3 +7,6 @@ npm-debug.log
 
 # Vim stuff
 *.swp
+
+# Ignore all snapshots
+*.png
```

Comment:
This will ignore *all* images. Probably not what you intended. Should probably only ignore the `snapshots/` directory.